Dockerfile

```diff
@@ -28,7 +28,7 @@ COPY src src
 RUN yarn build
 
 # Second stage, create slimmed down production-ready image
-FROM node:10
+FROM node:12
```

Comment:
Did you test it with node 12?

Reply:
I'm on it, thats why I created the PR with testmode :)

Reply:
just did and it worked with devmode :)

Reply:
(the CI also has node12 already btw)